.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+    -   id: check-case-conflict
+    -   id: check-json
+    -   id: mixed-line-ending
+    -   id: no-commit-to-branch
+        args: [--branch, staging, --branch, main, --branch, master, --branch, develop-4.0, --branch, develop-upstream]
+-   repo: https://github.com/codespell-project/codespell
+    rev: v2.2.2
+    hooks:
+    -   id: codespell
+        args: ["--count --ignore-words-list=ans,deriver,inout,packag --skip=\"*.js,*WordLists.swift,.git,Carthage,.build,build\""]
+# SwiftLint is commented out until we fix all SwiftLint warnings and errors.
+# https://github.com/web3swift-team/web3swift/pull/756
+#-   repo: https://github.com/realm/SwiftLint
```

Comment:
Should this be [`swiftlint --fix`](https://github.com/realm/SwiftLint#auto-correct) so that humans do not spend too much time placating the linter?

Reply:
Initially, I thought it would be better if developers would manually and consciously call `swiftlint --fix ...` on the files that are broken.
But now after I've seen the changes that `swiftlint --fix` produces and additionally I do not think most of the developers will look through the code to understand what actually has changed it would probably make sense to call this command with `--fix` argument as well.

I'll take some more time to think about it.

Reply:
Added `--fix` flag.